### ISSUE TITLE
migrate regression/product-details to cypress

### ISSUE THREAD

**Issue by Xymmer** [SAP/cloud-commerce-spartacus-storefront#1128]
See #1102 for original spike.
Take a look at big happy path for example.
See cypress video in our wiki.
Likely doesn't have login but if it needs login, set this to FR2 (and do after #1216).

all epic/cypressmigration tickets should be merged into epic branch "epic/cypressmigration-fr1".
if this epic branch doesn't exist, you're probably the first, so create it.

**Comment by Pucek9** [comment]
Skipped one test (should contain tabs with correct text) since #1408 is not resolved
